src/popup.html

```diff
@@ -7,37 +7,28 @@
     <link rel="stylesheet" type="text/css" href="css/popup.css" />
   </head>
   <body>
-    <h3>Notes App</h3>
-    <div class="alerts"></div>
-
-    <div class="container">
-
-
-      <nav>
-        <div class="nav nav-tabs" id="nav-tab" role="tablist">
-          <a class="nav-link active" id="nav-page-notes-tab"
-             data-bs-toggle="tab" href="#nav-page-notes" role="tab"
-             aria-controls="nav-page-notes" aria-selected="true">Current Page</a>
-          <a class="nav-link" id="nav-all-notes-tab"
-             data-bs-toggle="tab" href="#nav-all-notes" role="tab"
-             aria-controls="nav-all-notes"
-             aria-selected="false">All Notes</a>
+    <nav class="navbar navbar-expand-sm navbar-dark bg-dark">
+      <a class="navbar-brand" href="#">Note Taker</a>
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+        <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+          <ul class="navbar-nav">
+            <li class="nav-item">
+              <a class="nav-link active" aria-current="page" href="popup.html">Current Page</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" aria-current="page" href="allnotes.html">All Notes</a>
+            </li>
+          </ul>
         </div>
-      </nav>
-
-      <div class="tab-content" id="nav-tab-content">
-        <div class="tab-pane fade show active" id="nav-page-notes" role="tabpanel"
-             aria-labelledby="nav-page-notes-tab">
-          <textarea id="notes-content" class="form-control"></textarea>
-          <button id="save-button" type="button" class="btn-primary">Save</button>
-        </div>
-        <div class="tab-pane fade show active" id="nav-all-notes" role="tabpanel"
-             aria-labelledby="nav-all-notes-tab">
-          <div id="notes-urls"></div>
-        </div>
-      </div>
+    </nav>
 
+    <div class="container" id="main__add__section">
```

Comment:
Just curious, what naming convention is this using?

Reply:
This is definitely a mistake, I'll make sure one of goes through our HTML files soon and we can decide on a naming convention and change everything to stick to that. For now it is a little unorganized.